src/core/Akka/Actor/ActorCell.ReceiveTimeout.cs

```diff
@@ -19,12 +19,20 @@ public void SetReceiveTimeout(TimeSpan? timeout=null)
             _receiveTimeoutDuration = timeout;
         }
 
+        public TimeSpan? ReceiveTimeout
+        {
+            get
+            {
+                return _receiveTimeoutDuration;
+            }
+        }
+
```

Comment:
ReceiveTimeout getter is available in the Scala code base (together with SetReceiveTimeout) so we are inline with them.
This property is needed in the `TestActorRef_must_set_ReceiveTimeout_to_None` test (also ported from Scala)